garcon/activity.py

```diff
@@ -320,8 +320,9 @@ def run(self, identity=None):
                     self.fail(reason=str(error)[:255])
                     if self.on_exception:
                         self.on_exception(self, error)
-                except:
-                    pass
+                except Exception as error2:  # noqa: E722
```

Comment:
Is there a way to know the workflow has timed out before calling the `self.fail`?

Reply:
Unfortunately we cannot, execute_activity() method will even return successful response like everything went OK, so it's basically impossible to know if activity timed out.